pylint/config/config_file_parser.py

```diff
@@ -60,6 +61,15 @@ def _parse_ini_file(file_path: Path) -> tuple[dict[str, str], list[str]]:
                 options += [f"--{opt}", value]
         return config_content, options
 
+    @staticmethod
+    def _ini_file_with_sections(file_path: str) -> bool:
+        """Return whether the file uses sections."""
+        if "setup.cfg" in file_path:
+            return True
+        if "tox.ini" in file_path:
+            return True
+        return False
```

Comment:
```suggestion
        return any(n in file_path for n in ["setup.cfg", "tox.ini"])
```

Reply:
Let's leave this as is.

I can imagine we want to expand this. For example, by checking if a section starts with `tool:pylint` if we are in a `setup.cfg` file.